ibm/mas_devops/roles/suite_rollback/README.md

```diff
@@ -0,0 +1,78 @@
+suite_rollback
+===============================================================================
+This role validates if a given MAS installation is ready for the core platform to be rolled back to a specific MAS core version, and (as long as dry run mode is not enabled) will execute the rollback.
```

Comment:
Is there a level that this won't work at? i.e. MAS 8.9.? Is the user allowed to rollback minor levels as well as patch levels? Might need some guidance as to what is expected i.e. Can I rollback 8.11.1 to 8.9.4?

Reply:
Updated the Readme with more details.